rclpy/package.xml

```diff
@@ -14,9 +14,13 @@
   <depend>rmw_implementation</depend>
   <depend>rcl</depend>
 
+  <exec_depend>ament_index_python</exec_depend>
+
   <test_depend>ament_cmake_nose</test_depend>
+  <test_depend>ament_index_python</test_depend>
   <test_depend>ament_lint_auto</test_depend>
   <test_depend>ament_lint_common</test_depend>
+  <test_depend>launch</test_depend>
```

Comment:
This might become a problem in the future once `launch` will try to use `rclpy` to start the system and looking into the state of the system to e.g. to deterministic life cycle based start up.